webpack.umd.config.js

```diff
@@ -24,6 +24,7 @@ const webpackUMDConfig = {
     'react-dom': 'ReactDOM',
   },
   plugins: [
+    new webpack.DefinePlugin(config.compiler_globals),
```

Comment:
Okay, I'll remove it, but it strange - it was introduced in your [commit](https://github.com/Semantic-Org/Semantic-UI-React/pull/731/commits/4a69508f9187d69c0c8d2a260030e17ed63f6897).